nginx_stage/lib/nginx_stage.rb

```diff
@@ -81,6 +81,20 @@ def self.title(default: "Open OnDemand")
     end
   end
 
+  # The maximum file upload size that nginx will allow from clients in bytes
+  # @ example No maximum upload size supplied
+  #   nginx_file_upload_max #=> "10737420000"
+  # @ example 20 gigabyte file size upload limit
+  #   nginx_file_upload_max #=> "21474840000"
+  # @return [String] Maximum upload size for nginx
+  def self.upload_max(default: "10737420000")
+    upload_max = nginx_file_upload_max
+    if nginx_file_upload_max.empty?
```

Comment:
I initially used `nginx_file_upload_max: 21474840000` an integer here, and got this error.  Do we want to account for the case where users configure the integer and the a string?

```bash
Error -- undefined method `empty?' for 21474840000:Integer
Run 'nginx_stage --help' to see a full list of available command line options.
```